database/seed.js

```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+const db = require('./index.js');
+
+mongoose.connect('mongodb://localhost/opentable');
+
+const cuisines = ['American', 'Italian', 'Steakhouse', 'Seafood', 'French', 'Indian', 'Mexican', 'Japanese', 'Chinese', 'German', 'Spanish', 'Fusion/Eclectic', 'Barbecue', 'Greek', 'Grill', 'Comfort Food', 'Turkish', 'International', 'Thai', 'Tex-Mex', 'Bistro', 'Bar/Lounge', 'Californian'];
```

Comment:
consider breaking the array into multiple lines for readability

Reply:
Done